aiven/client/cli.py

```diff
@@ -2080,32 +2080,85 @@ def service__topic_get(self):
     @arg(
         "--operation",
         help="Task operation",
-        choices=["upgrade_check"],
+        choices=["migration_check", "upgrade_check"],
         default="upgrade_check",
     )
     @arg(
-        "--target_version",
+        "--target-version",
         help="Upgrade target version",
-        choices=["9.5", "9.6", "10", "11"],
+        choices=["10", "11", "12", "13"],
```

Comment:
I'd prefer using `aiven.shared.magic.PG_VERSIONS` here to avoid a stale in-place list in the future

```suggestion
        choices=PG_VERSIONS[1:],
```